apps/stops/lib/nearby.ex

```diff
@@ -110,7 +110,7 @@ defmodule Stops.Nearby do
   def do_routes_for_stop_direction(_, _), do: :error
 
   @spec merge_routes(String.t(), fun()) :: {:ok | :error, [route_with_direction] | :timeout}
-  defp merge_routes(stop_id, routes_fn) do
+  def merge_routes(stop_id, routes_fn) do
```

Comment:
If this is now a public function used elsewhere it would probably be good to have tests covering it.